### PR TITLE
Events

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this PR I added the logic and events of the calculator.
I added as well styles to maintain layout requirements.
Thank you for the review.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,47 +1,59 @@
...
-    this.state = {};
+    this.state = {
+      total: null,
+      next: null,
+    };
+    this.handler = this.handler.bind(this);
+  }
+
+  handler(e) {
+    this.setState((state) => calculate(state, e.target.innerText));
```
Comment (dasileker) [inline src/components/Calculator.js:16]: - [ ] The handler is failing while doing an operation on the `divide`.

Capture web_19-10-2021_151428_localhost

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Leandro-Barretoo,** 

### **TO Highlight**

- The PR is good.
- The commits are good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Leandro-Barretoo** [comment]
Thank you for spotting that problem.
So I fixed the problem and I hope everything looks good now.
Thank you for the review.

**Comment by Davidosky007** [comment]
## Hi @Leandro-Barretoo 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

done-gif


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**